website/client/js/controllers/rootCtrl.js

```diff
@@ -6,6 +6,9 @@
 habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$http', '$state', '$stateParams', 'Notification', 'Groups', 'Shared', 'Content', '$modal', '$timeout', 'ApiUrl', 'Payments','$sce','$window','Analytics','TAVERN_ID',
   function($scope, $rootScope, $location, User, $http, $state, $stateParams, Notification, Groups, Shared, Content, $modal, $timeout, ApiUrl, Payments, $sce, $window, Analytics, TAVERN_ID) {
     var user = User.user;
+    var IGNORE_SCROLL_PAGES = {
+        'options.social.challenges.detail': true
```

Comment:
Closing the challenge again causes it to jump up, so we'll probably need to add some additional state names here. I _think_ `options.social.challenges` should do it.